### PR TITLE
feat: add sample datasets and fully functional demos

### LINKED ISSUE #67: Add sample datasets & demos
It would be nice to have 'golden' datasets that can be used for demos that run out of the box or on github pages.

### PR DESCRIPTION
**Pull request by greghutch**
Sample datasets will eventually be used for regression testing.
Demos can be directly run using 'serve ...'. For now, API key will
need to be passed in manually on the url.

See live demo (now linked from the main readme): https://greghutch.github.io/fleet-debugger-demos/jump/

Changes:
 - Added link to to a github pages site where the demos
   are runnable.  Will work on hosting these on the main
   google-maps page (where hopefully we can use built-in
   api key magic).
 - added simple util-script to clean up datasets before
   publishing (removes project id, etc)
 - switched rawData.js to UMD so that it works in both
   react and nodejs

Fixes #67

### CODE REVIEW

File: src/vehicleData.js [hunk 1]
```diff
@@ -7,12 +7,12 @@
...
-const rawLogs = _.reverse(parsedJsonData.rawLogs);
-const jwt = parsedJsonData.jwt;
-const projectId = parsedJsonData.projectId;
-const apikey = parsedJsonData.APIKEY;
-const solutionType = parsedJsonData.solutionType;
+import { parsedData } from "./rawData";
+const rawLogs = _.reverse(parsedData.rawLogs);
+const jwt = parsedData.jwt;
+const projectId = parsedData.projectId;
+const apikey = parsedData.APIKEY;
```
Comment (ewbao) [inline src/vehicleData.js:14]: Is the capitalization here a naming pattern?

**Review by ewbao — COMMENTED** [review]
Do we need a valid API key and/or jwt to run the demo, since we're deleting those from the cleaned datasets?

File: src/vehicleData.js (hunk 1 above)
Comment (greghutch) [reply src/vehicleData.js:14]: Clearly there are consistency issues.  This repo is setup to enforce most formatting rules with eslint.  But I don't have something similar to enforce naming.    I think APIKEY is the odd-one out,  I'll file an issue to look at cleaning it up (https://github.com/googlemaps/fleet-debugger/issues/69)

**Comment by greghutch** [comment]
> Do we need a valid API key and/or jwt to run the demo, since we're deleting those from the cleaned datasets?

The demos are static html (with all the data baked in).  An API key is necessary to avoid an ugly watermark ... but the demos are still functional.   I'll sync up with our devrel folks on getting rid of the  watermark (it should be possible).

**Review by ewbao — APPROVED** [review]

**Pull request merged**